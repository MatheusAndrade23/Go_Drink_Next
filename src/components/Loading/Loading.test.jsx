import { screen } from '@testing-library/react';
import { renderTheme } from '../../tests/render-base.js';
import { Loading } from '.';

describe('<Loading />', () => {
  it('should render', () => {
    const { container } = renderTheme(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
