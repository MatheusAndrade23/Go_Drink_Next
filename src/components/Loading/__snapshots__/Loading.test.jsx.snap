// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should render 1`] = `
.c0 {
  position: absolute;
  left: 0;
  right: 0;
  height: 50vh;
  width: 100%;
  z-index: 100;
  background-color: transparent;
}

.c0:before,
.c0:after {
  content: '';
  position: absolute;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  border-width: 0.5rem;
  border-color: transparent;
  border-style: solid;
  border-radius: 50%;
}

.c0:after {
  width: 6rem;
  height: 6rem;
  border-left: 0.5rem solid #00ADB5;
  border-top: 0.5rem solid #00ADB5;
  -webkit-animation: cfKagt 600ms linear infinite;
  animation: cfKagt 600ms linear infinite;
}

.c0:before {
  width: 2rem;
  height: 2rem;
  border-left: 0.5rem solid #00ADB5;
  border-top: 0.5rem solid #00ADB5;
  -webkit-animation: cfKagt 1s linear reverse infinite;
  animation: cfKagt 1s linear reverse infinite;
}

<div>
  <span
    class="c0"
  />
</div>
`;
